app/routes/editor/problems/index.tsx

```diff
@@ -103,5 +121,12 @@ export default function EditorProblemsIndex() {
                 </Table>
             </TableContainer>
         </Box>
+
+        <PaginationComponent 
+            url={`/editor/problems?selection=${selection.toLowerCase()}`}
```

Comment:
If we will use `URLSearchParams` then we can basically pass `url` from loader's data and then parse it